src/config/config.ts

```diff
@@ -0,0 +1,15 @@
+export interface ConfigInterface {
+  port: number;
+  dbHost: string;
+  dbUser: string;
+  dbPassword: string;
+  dbPort: number;
+  dbForceSSL: boolean;
+  a0Enabled: boolean;
+  a0Domain?: string;
+  a0Audience?: string;
+  sentryEnabled: boolean;
+  sentryDsn?: string;
+};
+
+export const throwError = (message: string): never => { throw new Error(message); };
```

Comment:
This can also be used by the modules, but I am unsure how to share it. It's a weird hack to resolve an issue in Typescript's type resolving system and shouldn't need to exist, but I'm also wary of shoving it onto `global` scope.